src/mlpack/tests/q_learning_test.cpp

```diff
@@ -1,6 +1,6 @@
 /**
  * @file q_learning_test.hpp
- * @author Shangtong Zhang
+ * @author Shangtong Zhang and Rohan Raj
```

Comment:
You can just add a second line `@author Rohan Raj` here; I think Doxygen will not parse multiple authors on one line correctly.

Reply:
Should it be like?
@author Shangtong Zhang
@author Rohan Raj

Reply:
Right, exactly.

Reply:
I am waiting for this file to be reviewed . Once it is done, I will add all the changes accordingly. Thanks